ydb/core/fq/libs/config/protos/compute.proto

```diff
@@ -17,10 +17,22 @@ message TSynchronizationService {
 message TInPlaceCompute {
 }
 
+message TLoadControlConfig {
+    bool Enable = 1;
+    string MonitoringRequestDelay = 2;     // default 0 == non-stop polling
+    string AverageLoadInterval = 3;        // default "10s", allowed min "1s"
+    uint32 MaxClusterLoadPercentage = 4;   // default 0 == no load control
+    uint32 DefaultQueryLoadPercentage = 5; // default 10
+    uint32 PendingQueueSize = 6;           // default 0 == instant decline if overloaded
+    bool Inherit = 7;                      // whether to apply global config to databases (override, NOT merge)
+    uint32 CpuNumber = 8;
```

Comment:
Do we need this option? What does this option mean (per house, per cluster)?

Reply:
It is number if CPU per cluster. It is needed to align two different "systems". Why I ask for cluster load with Monitoring Service, I've got ratio between 0.0 and 1.0 for complete cluster. Query reports its usage in number of CPUs. To convert one to another we need to know number of CPU per cluster.

Reply:
Do we have another way to get this information? It's difficult to keep in consistent state two configs (ydb sls and fq proxy config). If we don't have another way than this solution is ok for me :)

Reply:
No other sources is available. This param is not mandatory, it is required for "cashback" feature - decrease CPU usage estimation after query completion. Allows better CPU utilization for short light queries